docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -0,0 +1,175 @@
+# Autofill Manager
+
+The Tizen autofill framework allows more than one or more autofill services to be installed on a Tizen device. Autofill Manager is a module for managing the installed autofill services. Application developers can use Autofill Manager to get the current autofill service or change the autofill service.
```

Comment:
Is "more than one or more" needed? How about just "one or more"?

Reply:
@mijin85cho, it is a good suggested change:
"one or more" can be used.
@imfine98, please incorporate the change.

Reply:
Fixed.